src/api/network/asio/Acceptor.cpp

```diff
@@ -108,23 +108,23 @@ struct Acceptor::Imp {
         }
 
         auto lock = Lock{lock_};
-        const auto bytes = [&] {
-            const auto address = next_socket_.remote_endpoint().address();
 
-            if (address.is_v4()) {
-                const auto bytes = address.to_v4().to_bytes();
-                auto* it = reinterpret_cast<const std::byte*>(bytes.data());
+        Space space{};
+        const auto address = next_socket_.remote_endpoint().address();
 
-                return Space{it, std::next(it, bytes.size())};
-            } else {
-                const auto bytes = address.to_v6().to_bytes();
-                auto* it = reinterpret_cast<const std::byte*>(bytes.data());
+        if (address.is_v4()) {
+            const auto bytes = address.to_v4().to_bytes();
+            auto* it = reinterpret_cast<const std::byte*>(bytes.data());
 
-                return Space{it, std::next(it, bytes.size())};
-            }
-        }();
+            space = Space{it, std::next(it, bytes.size())};
+        } else {
+            const auto bytes = address.to_v6().to_bytes();
+            auto* it = reinterpret_cast<const std::byte*>(bytes.data());
+
+            space = Space{it, std::next(it, bytes.size())};
+        }
         auto endpoint = opentxs::network::asio::Endpoint{
-            endpoint_.GetType(), reader(bytes), endpoint_.GetPort()};
+            endpoint_.GetType(), reader(space), endpoint_.GetPort()};
```

Comment:
why space instead of bytes, and why bytes are in v4 instead v6?

Reply:
This is just name, but I ok, bytes seems more relevant. I do not understand second part of your question though

Reply:
ok i saw it badly everytohng is ok